src/main/scala/beam/router/r5/R5RoutingWorker.scala

```diff
@@ -170,13 +169,44 @@ class R5RoutingWorker(workerParams: WorkerParameters) extends Actor with ActorLo
 
   var workAssigner: ActorRef = context.parent
 
+  private var maybeTravelTime: Option[TravelTime] = None
+
+  private var transitSchedule: Map[Id[BeamVehicle], (RouteInfo, Seq[BeamLeg])] = initTransitMap(isRemote)
+
+  def initTransitMap(isRemote: Boolean): Map[Id[BeamVehicle], (RouteInfo, Seq[BeamLeg])] = {
```

Comment:
Move this inside the config constructor. Then pass the inited map into the WorkerParameters